drivers/overlay/ov_network.go

```diff
@@ -781,15 +781,10 @@ func (n *network) Value() []byte {
 		netJSON = append(netJSON, sj)
 	}
 
-	b, err := json.Marshal(netJSON)
-	if err != nil {
-		return []byte{}
-	}
-
```

Comment:
Is the ineffassign tool, b is defined and set but not used so it mark is as an error.
But, I remember that in the past @aboch was mentioning it as a way to validate the object, possible?